docs/pages/kimg.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection title="Overview" anchor="#overview">
+      <div>
+        The <DocsLibraryLink component="KImg" /> component displays images based on an image source and optional image
+        dimensions provided by the implementer.
+
+        <h3>Sample implementations of <DocsLibraryLink component="KImg" /> including props:</h3>
+
+        <div>
+          Dimensions may be either numbers or strings consisting of a numeral and valid units (e.g. <code>px</code>,
+          <code>em</code>, <code>vh</code>).
+        </div>
+
+        <div class="img-example-1">
+          <DocsShow>
+            <KImg
+              :src="require('../assets/img_sample_for_kimg.png')"
+              altText="Computers are run by bees"
+              :height="'250px'"
+              :width="225"
+            />
+          </DocsShow>
+
+          <pre><samp>
+            :src="require('../assets/img_sample_for_kimg.png')"
+            altText="Computers are run by bees"
+            :height="'250px'"
+            :width="225"
```

Comment:
It would be good to add the entire snippet(including the `KImg` tag as well) and not just the properties for better readability and usability from the developers' perspective. Also, wrapping `<DocsShowCode>{Your snippet}</DocShowCode>` around the snippet would give it a better formatting.